SIWA guide (#7972)
auth0/docs: articles/connections/social/apple.md

## 1. Set up your app in your Apple Developer Account

Once your Developer Account is set up, you can follow the instructions in the [Apple Setup Guide](/connections/apple-setup) to get your app set up.

While setting up your app, make sure you save the following items for later:

* The Client ID (the Service ID)
* The Client Secret Signing Key
* The Apple Team ID
* The Client Signing Key ID (optional)

> Also note that when setting up the app, your **Return URL** should be in the following format: `https://<YOUR CUSTOM DOMAIN>/login/callback`. If you have any trouble, the [Apple Documentation](https://developer.apple.com/sign-in-with-apple/get-started/) may be of some use.

Reviewer: ```suggestion
When setting up the app, your **Return URL** should be in the following format: `https://<YOUR CUSTOM DOMAIN>/login/callback`. If you have any issues, refer to the [Apple Documentation](https://developer.apple.com/sign-in-with-apple/get-started/).
```

## 2. Create and enable a connection in Auth0

> Once you have the credentials you need from your Apple developer account, navigate to the [Connections > Social page](${manage_url}) in the Auth0 Dashboard, and click on the Apple connection.

Reviewer: ```suggestion
Once you have the credentials you need from your Apple developer account, go to [**Connections** > **Social**](${manage_url}) in the Dashboard, and click on the **Apple** connection.
```
